libvast/include/vast/pattern.hpp

```diff
@@ -27,11 +27,11 @@ class pattern {
   friend access;
 
 public:
-  struct options {
+  struct pattern_options {
```

Comment:
At this point it doesn't make sense to nest this struct inside `pattern` anymore. I'd move it to the top level.

Reply:
Makes sense. I moved `pattern_options` to the top level in https://github.com/tenzir/vast/pull/2964/commits/ca877c9f56a24e314666d51814b7e9064d30e5b8